game-core/src/main/java/games/strategy/triplea/ui/menubar/ExportMenu.java

```diff
@@ -390,9 +389,10 @@ private void addExportUnitStats() {
         return;
       }
       try (Writer writer = Files.newBufferedWriter(chooser.getSelectedFile().toPath(), StandardCharsets.UTF_8)) {
-        writer.write(
-            HelpMenu.getUnitStatsTable(gameData, uiContext).replaceAll("<p>", "<p>\r\n").replaceAll("</p>", "</p>\r\n")
-                .replaceAll("</tr>", "</tr>\r\n").replaceAll(LocalizeHtml.PATTERN_HTML_IMG_TAG, ""));
+        writer.write(HelpMenu
+            .getUnitStatsTable(gameData, uiContext)
+            .replaceAll("</?p>|</tr>", "$0\r\n")
+            .replaceAll("(?i)<img[^>]+/>", ""));
       } catch (final IOException e1) {
```

Comment:
Note to myself: This class does a lot of StringBuilding, some of which could definitely optimized by actively using RegEx features instead of chaining multiple replaceAll calls.
Also there are a lot of places where 1-character strings are being appended to StringBuilders, which could be slightly optimized by using primitive chars instead. The Garbage collector will thank us.